std_msgs/Header header
uint32 f1
uint32 f2
geometry_msgs/Pose rel_pose